components/rmrk/Profile/NavbarProfileDropdown.vue

```diff
@@ -45,8 +45,8 @@
           <nuxt-link to="/transform">{{ $t('transform') }}</nuxt-link>
         </b-dropdown-item>
       </template>
-      <b-dropdown-item v-if="isBsx" has-link aria-role="menuitem">
-        <nuxt-link to="/bsx/incomingoffers">{{
+      <b-dropdown-item v-if="isBsx || isSnek" has-link aria-role="menuitem">
```

Comment:
can we make this just 1 prop called "showIncommingOffers"
and in `Navbar.vue` just propagate it via `:show-incomming-offers="isBsx || isSnek"`